### PR TITLE
add extra functionality to channellock (instant unlock), fix import errors and formatting, etc

### PR DESCRIPTION
**Pull request by Rahul-Maru**
…hannellock() to checklocks.py, fix some import errors, minor formatting changes

### CODE REVIEW

**Comment by intfract** [comment]
Changes for `src/on_ready.py:8`:
```py
await bot.change_presence(activity=discord.Game(name="u_int"))
```

Changes for `src/app.py`:
- use 4 spaces for indenting not 2
- stop using aligning function parameters because it wastes a lot of whitespace and makes everything look weird

```py
async def lockcommand(
    interaction: discord.Interaction,
    channelinput: discord.TextChannel = discord.SlashOption(
        name="channel_name",
        description="Which channel do you want to lock?",
        required=True
    ),
    # other parameters
):
```

File: src/app.py [hunk 1]
```diff
@@ -1637,7 +1642,7 @@ async def lockcommand(interaction: discord.Interaction,
   await logchannel.send(f"Channel Name: {channelid}\n" f"Lock Time: {locktime} ({locktimeinunix})\n" f"Unlock Time: {unlocktime} ({unlocktimeinunix})\n")
 
   client = pymongo.MongoClient(LINK)
-  db = client.IGCSEBot
+  db = client.rigcse
```
Comment (Sachin-dot-py) [inline src/app.py:1645]: Switch this back to IGCSEBot before merging

File: src/app.py [hunk 2]
```diff
@@ -1608,6 +1599,20 @@ async def lockcommand(interaction: discord.Interaction,
                                 ephemeral=True)
         return
 
+  # effectively resets channellock database. for developer purposes || TODO make this work
+  if locktime == "resolveall" and unlocktime == "!@#$%^&*()":
+    client = pymongo.MongoClient(LINK)
+    db = client.rigcse
```
Comment (Sachin-dot-py) [inline src/app.py:1605]: Switch this back to IGCSEBot before merging

File: src/checklocks.py [hunk 3]
```diff
@@ -1,11 +1,30 @@
...
+      # If the channel was locked, send another embed with unlock time
+      embed = discord.Embed(description=f"Unlocking channel <t:{unlocktime}:R>.")
+      await channel.send(embed=embed)
 
 @tasks.loop(seconds=60)
 async def checklocks():
     """Checks the database every 60 seconds to see if anything needs to be locked or unlocked """
     client = pymongo.MongoClient(LINK)
-    db = client.IGCSEBot
+    db = client.rigcse
```
Comment (Sachin-dot-py) [inline src/checklocks.py:27]: Switch this back to IGCSEBot before merging

File: src/on_ready.py [hunk 4]
```diff
@@ -5,7 +5,7 @@
 @bot.event
 async def on_ready():
     print(f"Logged in as {str(bot.user)}.")
-    await bot.change_presence(activity=discord.Game(name="u_int"))
+    await bot.change_presence(activity=discord.Game(name="Chesskid.com"))
```
Comment (Sachin-dot-py) [inline src/on_ready.py:8]: Switch to Watching r/IGCSE

File: src/checklocks.py [hunk 5]
```diff
@@ -1,11 +1,30 @@
...
-from bot import tasks, pymongo
+import time
+import traceback
+from constants import LINK, GUILD_ID
+from bot import tasks, pymongo, discord, bot
+from roles import get_role
+
+async def togglechannellock(channelid, unlock, *, unlocktime=0):
+    """Function for locking/unlocking a discord channel"""
+    everyonerole = await get_role("@everyone")
```
Comment (Sachin-dot-py) [inline src/checklocks.py:9]: Does this work? `guild.default_role` also gives you the `@everyone` role object

**Review by Sachin-dot-py — COMMENTED** [review]
A few changes before merging (added comments)

**Pull request closed without merging**